spec/schemas/query._common.yaml

```diff
@@ -79,10 +79,10 @@ components:
             $ref: '#/components/schemas/RootCause'
         type:
           type: string
-          example: status_exception
+          # example: status_exception
```

Comment:
Commented out because fails with strict mode as follows:
`Error: strict mode: unknown keyword: "example"`

Reply:
Open an issue for these?